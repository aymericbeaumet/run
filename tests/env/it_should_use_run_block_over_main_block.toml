env = [ "__RUN_CLI_FOOBAR__=from_main_block" ]

[[run]]
env = [ "__RUN_CLI_FOOBAR__=from_run_block" ]
name = "p3"
cmd = ["printenv", "__RUN_CLI_FOOBAR__"]
