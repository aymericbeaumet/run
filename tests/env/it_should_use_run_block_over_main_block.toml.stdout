[p3] from_run_block